coala_quickstart/interaction/Question.py

```diff
@@ -0,0 +1,23 @@
+def ask_question(printer, question, default=None):
+    """
+    Asks the user a question and returns the answer.
+
+    :param printer:
+        A ``ConsolePrinter`` object used for console interactions.
+    :param question:
+        The question text.
+    :param default:
+        A default placed next to the question; used as the default
+        value if the response is empty.
+    """
+    print("")
```

Comment:
either use the printer consistently or print, it could be a printer that does some weird stuff like printing everything to stderr or so